python/cuml/test/test_pickle.py

```diff
@@ -303,6 +299,22 @@ def assert_model(pickled_model, X_test):
     pickle_save_load(tmpdir, create_mod, assert_model)
 
 
+@pytest.mark.parametrize('model_name',
+                         all_models.keys())
+def test_unfit_pickle(model_name):
+    # Any model xfailed in this test cannot be used for hyperparameter sweeps
+    # with dask or sklearn
```

Comment:
This should be in the docstring of their classes (not sure if we have a note anywhere else regarding PCA and UMAP on this, also we should probably open github issues (if there exist none) and put them in the hyperparam board

Reply:
Yes, good point. Filed as Issue #1538

Reply:
Ok, this was pretty easy to fix rather than commenting. Thanks for pointing it out. Fixed in the updates.